scripts/build_provenance

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+readonly SCRIPTS_DIR="$(dirname "$0")"
+# shellcheck source=scripts/common
+source "$SCRIPTS_DIR/common"
+
+readonly BUILD_SCRIPT=$1
+readonly OUTPUT_PATH=$2
+
+readonly BUILD_COMMAND="./scripts/docker_run ${BUILD_SCRIPT}"
```

Comment:
This seems to tie the provenance to docker_run specifically, i.e. it would only be reproducible if the target repo actually has this script, right? Maybe it's more a question for the transparent release reproduce-build step, but I feel I'm still missing something in how these are related. Perhaps it may be easier to reason about once we generate the SLSA provenance instead of the TOML file?